.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: minimal
+
+services:
+  - docker
+
+stages:
+  - build_and_docker_push
+  - docker_pull_and_test
+
+jobs:
+  include:
+    - stage: build_and_docker_push
+      script: ./travis-stage.sh build && ./travis-stage.sh docker_push
+    - stage: docker_pull_and_test
+      env: TOXENV=py39 MODULEMD_TOOL=repo2module SITEPACKAGES=true
```

Comment:
Right, the other tests can be simply added as another TOXENV e.g. with different python versions, if necessary. And, to have the outputs split for each tool, it's better to have each toxenv as a separate job where we use a pre-built image. That's also an answer to your general comment. It would speed up things and would help distinguish where the problem actually occurred, than have all the tools tested together in one job. Wdyt?

Here's how the pipeline looks like, if there's an image built first:
https://travis-ci.com/github/fivaldi/modulemd-tools/builds/212806308

I think for this _special_ python` project repo layout with multiple projects inside, it's better to have one tox than to maintain several configs for each tool. I don't know...